csfieldguide/chapters/content/en/software-engineering/sections/design.md

```diff
@@ -119,12 +111,12 @@ In this project, we will look at how to design the software.
 Start by thinking about how the software you are trying to build can be broken up into smaller parts.
 Maybe there is a database or a user interface or a website?
 For example, imagine you are writing software to control a robot.
-The robot needs to use its sensors to follow a black line on the ground until it reach a target.
+The robot needs to use its sensors to follow a black line on the ground until it reaches a target.
 The software for your robot should have a part that interacts with the sensors to get information about what they "see".
 It should then pass this information to another part, which analyses the data and decides where to move next.
 Finally, you should have a part of the software which interacts with the robot’s wheels to make it move in a given direction.
 
-Try to break down your software into as many parts as possible (remember, small components are much easier to build!) but don’t go too far - each part should perform a sensible task and be relatively independent from the rest of the system.
+Try to break down your software into as many parts as possible (remember, small components are much easier to build!) but don’t go too far &ndash; each part should perform a sensible task and be relatively independent from the rest of the system.
 
 For each part that you have identified, write a brief description about what it does.
```

Comment:
(doesn't relate to this particular line btw just needed to comment somewhere)

Reply:
Do you have any suggestions on how to structure it? Is there a Markdown subtitle syntax we use?

Reply:
Hmmm no I don't.. Now that I think about it I don't think there is anywhere else in CSFG where we have a subtitle _or_ the little description after the colon in the heading.. I do like it but since this is the only place it occurs maybe we could remove it? And mention somewhere in the opening paragraph/sentence? Not sure ..

Reply:
![image](https://user-images.githubusercontent.com/25916475/57657971-33708d00-7631-11e9-833a-7a353a857bda.png)

It does happen elsewhere, even on the second-largest screen size we use, so I don't think it's something to work on in this PR. Make an issue for it?

Reply:
Sounds good